tests/PHPStan/Command/ErrorFormatter/data/windowsBaseline.neon

```diff
@@ -11,7 +11,7 @@ parameters:
 			path: UnixNewlines.php
 
 		-
-			message: "#^PHPDoc tag @param has invalid value \\(\\)\\: Unexpected token \"\\\\r\\\\n\\\\t \\*\", expected type at offset 110$#"
+			message: "#^PHPDoc tag @param has invalid value \\(\\)\\: Unexpected token \"\\\\r\\\\n     \\*\", expected type at offset 122$#"
```

Comment:
main difference I am targeting is the "at offset" 110 vs. 122 value.
I guess the 4-spaces vs. \t char is a different/unrelated problem, and can maybe even be reverted